src/test/java/org/killbill/billing/plugin/adyen/TestAdyenPaymentPluginApi.java

```diff
@@ -71,7 +71,7 @@ public void testCreatePaymentMethod() throws PaymentPluginApiException {
     Assert.assertEquals(1, syncPaymentMethods(kbAccountId).size());
```

Comment:
Nit: JUnit expects "expected" on the left side, and "actual" on the right side. It's the opposite with TestNG (https://javadoc.io/doc/org.testng/testng/latest/org/testng/Assert.html).

This should technically be `Assert.assertEquals(syncPaymentMethods(kbAccountId).size(), 1);` (all assertions should be modified 😓).

Reply:
Fixed